danger-undercover.gemspec

```
@@ -8,35 +9,36 @@ Gem::Specification.new do |spec|
   spec.version       = Undercover::VERSION
   spec.authors       = ['rafayet-monon']
   spec.email         = ['rafayet.monon@gmail.com']
```

For all projects, please:
- Add `dev@nimblehq.co` as the first email so that all devs can be reached 
- Use your personal NImble email since the repository is under the Nimble organization

```suggestion
  spec.authors       = ['Nimble', 'rafayet-monon']
  spec.email         = ['dev@nimblehq.co', 'rafayet@nimblehg.co']
```